src/services/processTx.js

```diff
@@ -108,7 +108,11 @@ async function newPendingTran(tx, protocol) {
             // smart contract call hence the asset must be the token name
             to = data.params[0].value;
             pillarId = await client.getAsync(to);
-            [, { value }] = data.params;
+            value = data.params[1].value;
+          } else if (data.name === 'transferFrom' || data.name === 'safeTransferFrom') {
+            to = data.params[1].value;
+            pillarId = await client.getAsync(to);
+            value = data.params[2].value;
```

Comment:
this one could be  [, ,  { value }] = data.params;

Reply:
Pls see earlier, will amend.